solutions/standard/DA-keys.md

```diff
@@ -0,0 +1,65 @@
+# Defining keys in a Key Management Service configuration in an IBM Cloud Project
+
+When adding Key Management Service from the IBM Catalog to an IBM Cloud project it is possible
+to configure additional key rings and keys. This is achieved by entering edit
+mode for the project configuration and selecting the `Configure` pane and the `optional` tab.
```

Comment:
```suggestion
When you add a key management service from the IBM Cloud catalog to an IBM Cloud Projects service, you can configure key rings and keys. In the edit mode for the projects configuration, select the Configure panel and then click the optional tab.
```

moving to single line to reduce possibility of newlines in the rendering and to make maintenance easier.

we're specific about panes vs panels vs windows vs boxes. I'm not sure which one this is
- A pane is a framed section of a window that provides different views of the same document or independent but related information.
- A panel sits flush to either the left or right side.
- A window is an area of the screen with visible boundaries that displays information. A window might comprise multiple panes.